mmhuman3d/data/datasets/human_pymafx_dataset.py

```diff
@@ -0,0 +1,310 @@
+import os
+import os.path as osp
+from abc import ABCMeta
+from typing import Optional
+
+import cv2
+import numpy as np
+import torch
+import torch.nn.functional as F
+from torchvision.transforms import Normalize
+
+from mmhuman3d.utils.img_utils import (
+    crop,
+    flip_img,
+    flip_kp,
+    get_transform,
+    transform,
+)
+from .base_dataset import BaseDataset
+from .builder import DATASETS
+
+HF_IMG_SIZE = 224
+
+IMG_RES = 224
+IMG_NORM_MEAN = [0.485, 0.456, 0.406]
+IMG_NORM_STD = [0.229, 0.224, 0.225]
```

Comment:
Shall we try to avoid using global variables?

Reply:
I deleted it, and added them to config file.